Build_Upgrade_Package.bat

```diff
@@ -30,15 +30,26 @@ goto finish
 @echo .
 @echo .-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-
 @echo .
-@echo . Building application using
+@echo . Creating upgrade package using:
 @echo .     "%builder%"
 @echo .
 @echo .-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-
 @echo .
 
-.nuget\NuGet.exe restore DNN_Platform.sln
-"%builder%" /t:CreateUpgrade /v:m Build/BuildScripts/DNN_Package.build
+pushd Artifacts
+del /f /q *Upgrade.zip
+popd
+pushd "DNN Platform"
+git clean -xdf
+popd
+pushd Website
+git clean -xdf
+popd
```

Comment:
I don't think I'm a fan of git-cleaning every single time. It takes quite a while and usually should not be necessary.

Applies to all scripts ofc

Reply:
Without cleaning, you will not receive clean artifacts and the produced zip files will contain some unnecessary files in them, especially the source code.